docs/ktx/firestore.md

```diff
@@ -67,4 +67,28 @@ val objectList = snapshot.toObjects(MyClass::class.java)
 ```kotlin
 val snapshot: QuerySnapshot = ...
 val objectList = snapshot.toObjects<MyClass>()
-```
\ No newline at end of file
+```
+
+### Setup Firestore with a local emulator
+
+**Kotlin**
+```kotlin
+val settings = FirebaseFirestoreSettings.Builder()
+                   .setHost("10.0.2.2:8080")
+                   .setSslEnabled(false)
+                   .setPersistenceEnabled(false)
+                   .build()
+
+FirebaseFirestore.getInstance().setFirestoreSettings(settings)
+```
+
+**Kotlin + KTX**
+```kotlin
+val settings = firestoreSettings {
+    setHost("10.0.2.2:8080")
+    setSslEnabled(false)
+    setPersistenceEnabled(false)
+}
+
+Firebase.firestore.setFirestoreSettings(settings)
```

Comment:
I'd suggest two alternatives/additions to consider:

**1. Pass settings when initializing firestore**
Context: it's only possible to set setting on firestore before it is used, so it may be a good idea to make it impossible/hard to do differently.

Namely by passing settings when initializing firestore, that way you only get to set the settings when you're creating it.

```kotlin
val firestore = Firebase.firestore {
  host = "10.0.2.2:8080"
  sslEnabled = false
}
val anotherFirestore = Firebase.firestore(app) {
  host = "10.0.2.2:8080"
  sslEnabled = false
}
```

It might make sense to do the same for the Java SDK, but that's a different story

**2. Make setter an extension to `FirebaseFirestore` to make things more concise**

```kotlin
Firebase.firestore.setSettings {
  host = "10.0.2.2:8080"
  sslEnabled = false
}
```

Reply:
@vkryachko I really like both additions. I've just added them.